third_party/Android.mk

```diff
@@ -85,13 +85,29 @@ include $(BUILD_STATIC_LIBRARY)
 
 SPVTOOLS_LOCAL_PATH := $(THIRD_PARTY_PATH)/spirv-tools
 LOCAL_PATH := $(SPVTOOLS_LOCAL_PATH)
+SPVTOOLS_OUT_PATH=$(abspath $(TARGET_OUT))
+
+define gen_spvtools_grammar_tables
+$(call generate-file-dir,$(SPVTOOLS_OUT_PATH)/opcode.inc)
+$(1)/opcode.inc $(1)/operand.inc: $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
+                                  $(SPVTOOLS_LOCAL_PATH)/source/grammar.json
+		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
+		                $(SPVTOOLS_LOCAL_PATH)/source/grammar.json \
+		                --opcode-file=$(1)/opcode.inc \
+		                --operand-file=$(1)/operand.inc
+		@echo "[$(TARGET_ARCH_ABI)] Grammar: opcode.inc & operand.inc <= grammar.json"
+$(SPVTOOLS_LOCAL_PATH)/source/opcode.cpp: $(1)/opcode.inc
+$(SPVTOOLS_LOCAL_PATH)/source/operand.cpp: $(1)/operand.inc
```

Comment:
Wow! That's something I was not aware of, but makes sense for me. Compilation works fine though.